ssf-1.1.0/README_WOLF.md

```diff
@@ -0,0 +1,114 @@
+# Building SSF with wolfSSL
+
+Port of SSF 1.1.0 with Boost/Asio 1.65.1 for wolfSSL. Project completed 07/2018.
+
+## Overview
+
+The SSF tool uses the Boot.Asio component for the TLS interface. WolfSSL has been ported into Boost.Asio and SSF builds. 
+
+The build option `ASIO_USE_WOLFSSL` is used to indicate wolfSSL as the TLS provider.
+
+## Getting Sources
+
+1. Clone the master version of wolfSSL with 
+
+    $ `git clone https://github.com/wolfSSL/wolfssl.git`
+    OR
+    $ `scp wolfssl-3.15.3.tar.gz ubuntu@204.48.22.68:~/ssfwolfbuild`
+    $ `tar -xzf wolfssl-3.15.3.tar.gz`
+    $ `mv wolfssl-3.15.3 wolfssl`
+
+2. Execute the following commands from the wolfSSL root directory
+
+    $ `cd wolfssl`
```

Comment:
Above it says to start in the wolfSSL root directory. Do we want to change directory into the headers directory here?